src/Calypso-SystemTools-Core/ClyMethodEditorCommand.class.st

```diff
@@ -17,9 +17,22 @@ Class {
 	#category : #'Calypso-SystemTools-Core-Editors-Methods'
 }
 
+{ #category : #testing }
+ClyMethodEditorCommand class >> canBeExecutedInCodeMethodEditor: aTool [
```

Comment:
somehow this method is duplicated in file

Reply:
no is not